scripts/question_answering/index.rst

```diff
@@ -0,0 +1,68 @@
+Question Answer
+-------------------
+

```

Comment:
Could you add a download link here? GluonNLP's build system will automatically create a zip file for the scripts. You can use the following code to create a link for that file.
```
:download:`[Download] </model_zoo/question_answering.zip>`
```

Reply:
Done.